docs/guided-tour/blueprints/echo-server/README.md

```diff
@@ -74,10 +74,10 @@ component. In detail, the connection is the following:
     - image: {{ .Values.image }}
   ```
   
-> **_NOTE:_** Since Kubernetes does not support OCM (yet ;) ), we need the *oci reference* of the container image, here.
-> Consequently, to actually use this component with landscaper, the container image that has to be deployed in a pod 
-> cannot be embedded into the component as a local blob (though it make sense to do so, as an intermediate step during 
-> transport of the component).
+> **_NOTE:_** Since Kubernetes does not support OCM, we need the *oci reference* of the container image, here.
+> Consequently, to actually use this component version with landscaper, the container image that has to be deployed in 
+> a pod cannot be embedded into the component as a local blob (though it make sense to do so, as an intermediate step 
+> during transport of the component).
```

Comment:
Does a reader really know what a local blob is in the context of ocm? I don't think so.

I think we need to give the user a more detailed description of how they have to specify OCI image resources in the component version so that they will be consumable by k8s. Could also be part of the ocm.software docs and be referenced here.

Reply:
We will rework the three blueprints example and combine them into one example. It is on our follow-up list.